akash1/HTML/ASSIn2/Assignment 2.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1 style="text-align: center">Google</h1>
+    <i>This article is about the company</i><br />
+    <h1>From the garage to the Googleplex</h1><br />
+    <span style="font-family: Verdana">
+      The Google story begin in 1995 at <b>Stanford University.</b><br />
+      <b><i>Larry Page</i></b> was considering Stanford for grad school and
+      <b><i>Sergey Brin</i></b>, a student<br />there, was assigned to show him around.
+    </span>
+    <p>
+      By some accounts, they disagreed about nearly everything during that first
+      meeting, but by the following<br />
+      year they struck a.<br />
+      Working from their dorm rooms, they built a search engine that used links
+      to determine the importance of<br />
+      individual pages on the <span style="color: blue">World Wide Web.</span>
+    </p>
+    <p>They called this search engine <i>Backrub.</i> <br /></p><hr/><br />
+    <h1>Initial public offering</h1>
+    <br />Google's initial public offering (IPO) took place five years later, on 19
+    <sup>th</sup> August 2004.<br />
+    At that time Larry Page. Sergey Brin, and Eric Schmidt agreed to work
+    together at Google for
+    <del>30</del> <u>20</u><br />years, until the year 2024.<p>&nbsp; Thank You!!</p>
+    <br /><br /><br />
+    <div style="border: 2px solid gray; background-color: indianred">
+      <h1 style="text-align: center">Google</h1>
+      <i>This article is about the company</i> <br />
+      <h1>From the garage to the Googleplex</h1><br />
+      <p>The Google story begin in 1995 at <b>Stanford University.</b><br /></p>
+      <b><i>Larry Page</i> </b> was considering Stanford for grad school and
+      <b><i>Sergey Brin,</i></b>a student there, was assigned to show him around
+      <p>
+        By some accounts, they disagreed about nearly everything during that
+        first meeting, but by the following year they struck a
+        <mark>parternship.</mark><br />
+        Working from their dorm rooms, they built a search engine that used
+        links to determine the importance of individual pages on the
+        <span style="color: blue">World Wide Web.</span>
+      </p>
+      <p>
+        They called this search engine
+        <span style="background-color: blue"><i>Backrub.</i></span> <br/>
+      </p>
+      <hr font color="blue" /><br />
```

Comment:
which type of style is this? remove if not necessary and dont use unnecessary hr and br tag